subsys/debug/gpio_frontend/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+zephyr_sources(gpio_frontend.c)
```

Comment:
This should be optimized for speed, not whatever the default is (probably size).

zephyr_library()
zephyr_library_sources(gpio_frontend.c)
zephyr_library_compile_options(${OPTIMIZE_FOR_SPEED_FLAG})